templates/zerver/emails/sponsorship_request.html

```diff
@@ -0,0 +1,30 @@
+{% extends "zerver/emails/email_base_messages.html" %}
+
+{% block content %}
+
+<b>Requested by</b>: {{ requested_by }}
+
+<br><br>
+
+<b>User role</b>: {{ user_role }}
```

Comment:
Is this field useful?  It should always be an admin, right?

Reply:
Oh, I see, it could be a billing admin.  OK.

Reply:
Yeah. Anyone can upgrade/requst for upgrade to Standard.

Should we mark the person who requested for sponsorship as billing_admin? In the normal upgrade, we do that but I am not sure whether that should be the case here. 

The code currently is written in a way that anyone can access /billing if the realm is pending sponsorship. The page just says the organization is pending sponsorship. Should we instead restrict this to only admin/billing admins?

Reply:
Yeah, let's only allow admins/billing admins to take actions here; seems cleaner that way.